src/module.py

```diff
@@ -3,22 +3,157 @@
 for preprocessing speech and text, the transformer & RNN encoder/decoder, & post
 processing modules for text and speech.
 '''
-# TODO: Consider adding pre/post net base class.
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import numpy as np
+from collections import OrderedDict
+
+from data.symbols import symbols
+
+class Linear(nn.Module):
+    """
+    Linear Module copied from Transformer-TTS.
+    
+    A linear layer initialized using the Xavier Uniform method.
+    """
+    def __init__(self, in_dim, out_dim, bias=True, w_init='linear'):
+        """
+        :param in_dim: dimension of input
+        :param out_dim: dimension of output
+        :param bias: boolean. if True, bias is included.
+        :param w_init: str. weight inits with xavier initialization.
+        """
+        super(Linear, self).__init__()
+        self.linear_layer = nn.Linear(in_dim, out_dim, bias=bias)
+
+        nn.init.xavier_uniform_(
+            self.linear_layer.weight,
+            gain=nn.init.calculate_gain(w_init))
+
+    def forward(self, x):
+        return self.linear_layer(x)
+
+class Conv(nn.Module):
```

Comment:
I am not a big fan of these modules, as they essentially just wrap the pytorch modules and use xaiver_uniform instead of Kaiming

Can we rewrite these pre/post net to just directly use the `nn.*` modules instead?  Or perhaps make that a TODO for later - regardless I think we need to consolidate on an approach and I prefer just using what pytorch has to offer over this wrapping approach

Reply:
This isn't really a blocker, so in the interest of making progress on the general training loop I am going to proceed with merging this

Reply:
I added these because it seems that the Postnets also use them. Nonetheless, I don't think it would make a big difference if we used Kaiming initialization instead of Xavier. @billyang98 what do you think?